packages/loader/package.json

```diff
@@ -35,8 +35,8 @@
     "loader"
   ],
   "scripts": {
-    "test-api": "jest test",
-    "test-coverage": "jest test --coverage",
+    "test-api": "NODE_OPTIONS=--experimental-vm-modules jest test",
+    "test-coverage": "NODE_OPTIONS=--experimental-vm-modules jest test --coverage",
```

Comment:
`--experimental-vm-modules` for [tests written in ESM](https://jestjs.io/docs/ecmascript-modules#:~:text=execute%20node%20with%20--experimental-vm-modules).